.github/workflows/backport.yml

```diff
@@ -1,17 +1,18 @@
 name: Backport
 on:
-  pull_request:
-    types: [closed]
+  pull_request_target:
```

Comment:
Noticing this event grants any scripts we execute write access to the repo. Since it doesn't run any scripts in the repo, should be all set :+1: